Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
+    dependencies: [
+        .Package(url: "https://github.com/groue/SQLiteSDK.git", majorVersion: 0)
+    ],
+    exclude: ["GRDB.xcworkspace", "Playgrounds", "SQLCipher", "SQLiteCustom", "Support", "DemoApps", "Tests", "Documentation"]
```

Comment:
Yes, we can move GRDB dir into Sources/

There are three possible layouts:

1) Dirs for each module in root (currently used). The downside is having to exclude everything which SPM doesn't know how to handle.

2) For single module repos, Sources/ can contain source files **without** directories.

3) If at least one directory is present in Sources/, dirs are treated as distinct modules and no other source files are allowed at top level in Sources/
We can use this layout. So, it will become:
Sources/GRDB/...other files...